index.html

```diff
@@ -180,14 +180,15 @@
         </li>
         <li>When the `type` attribute of the [link] element of a <a>preload
         link</a> that is already [in a Document] but was previously not
-        obtained due to the `type` attribute specifying an unsupported
+        obtained due to the `type` attribute not specifying a
+        [parsable MIME type] or specifying an [unsupported MIME type] for the
         <a>request destination</a> is set, removed, or changed.
```

Comment:
It could, but we don't really know the request type here (and there's no clear mapping from destination to type, e.g. for `media`)